src/tiles/grid/tile-stats.ts

```diff
@@ -0,0 +1,104 @@
+import { css, CSSResultGroup, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { favoriteFilledIcon } from '../../assets/img/icons/favorite-filled';
+import { reviewsIcon } from '../../assets/img/icons/reviews';
+import { uploadIcon } from '../../assets/img/icons/upload';
+import { viewsIcon } from '../../assets/img/icons/views';
+
+import { formatCount } from '../../utils/format-count';
+
+@customElement('tile-stats')
+export class TileStats extends LitElement {
+  @property({ type: String }) mediatype?: string;
+
+  @property({ type: Number }) itemCount?: number;
+
+  @property({ type: Number }) viewCount?: number;
+
+  @property({ type: Number }) favCount?: number;
+
+  @property({ type: Number }) commentCount?: number;
+
+  render() {
+    console.log('tile-stats here');
+    return html`
+      <div class="stats-wrapper">
```

Comment:
```suggestion
      <div class="item-stats">
```
suggestion: create more meaningful class names